### DISCUSSION TITLE
Some pages of a pptx can't be converted into pdf correctly(libreoffice)

### DISCUSSION THREAD

**Discussion by yongjinhuang** [gotenberg/gotenberg#1038] [Q&A]
<img width="1891" alt="image" src="https://github.com/user-attachments/assets/86052980-7698-490a-a873-836447599a49">
As the image shows, some pages of a PowerPoint can not be converted into PDF correctly. The part of these pages can't be copied too

**Comment by gulien** [comment]
Hello @YongjinHuang,

Does it work when you convert it with LibreOffice directly?

**Comment by yongjinhuang** [comment]
@gulien Thanks for answering. When I convert it with LibreOffice, it acts like what `gotenberg` does.

**Comment by gulien** [comment]
Thanks for the feedback! Not sure I can do much here, because its seems like an "issue" with LibreOffice itself. I'm converting this issue to a "discussion", maybe someone will provide more insight 🤷‍♂️
